goal_src/engine/load/loader.gc

```diff
@@ -991,7 +991,20 @@
             (str-play-async (-> arg0 name) spool-sound)
             (set! (-> *art-control* active-stream) (-> arg0 name))
             )
-          (let* ((f30-0 (* 0.05859375 (-> s2-4 speed)))
+          
+          (let* ((f30-0 (* (if (and (= (-> *setting-control* current video-mode) '150fps)
+                                    (or (string= (-> arg0 name) "eichar-ambient-1")
+                                        (string= (-> arg0 name) "eichar-ambient-2")
+                                        (string= (-> arg0 name) "eichar-ambient-3")
+                                        (string= (-> arg0 name) "eichar-ambient-4")
+                                        )
+                                    )
+                               0.4
+                               1.0
+                               )
+                           0.05859375 (-> s2-4 speed)
+                           )
+                   )
```

Comment:
It might be better to check if the spool is playing sound or not? But tbh I'm not sure how to properly check for that. While you're at it though, decompile `loader` and copy the new output for `ja-play-spooled-anim` and `ja-abort-spooled-anim` since it has the new `ja` macros that massively simplify things. I seem to have forgotten to do that before lol.

Reply:
Unfortunately the idles play sound with clapping/yawning

Reply:
What if you checked for something like `(= -1 (current-str-pos spool-sound))`?

Reply:
that counts cutscenes as well it seems

Reply:
checking `(= 1 (-> arg0 parts))` works well, but I think that it might also slow down some things like tips? Not bad though

Reply:
It might break some of the death cutscenes.

Reply:
yep it does